$NetBSD: patch-ad,v 1.1 2005/10/22 18:12:09 rillig Exp $

 - <locale.h> is needed to declare LC_ALL.
 - update const compliance for recent C++ compilers

--- main.cpp.orig	2007-02-10 15:08:27.000000000 +0000
+++ main.cpp
@@ -2,6 +2,7 @@
 #	pragma warning( disable : 4786 )		// stl warning ¾ø¾Ö±â
 #endif
 
+#include <locale.h>
 #include <stdio.h>
 #include <iostream>
 #include <time.h>
@@ -131,7 +132,7 @@ int main(int argc, char* argv[])
 
 	CUnAlz	unalz;
 	char* source=NULL;
-	char* destpath=".";
+	const char* destpath=".";
 	char* destcodepage=NULL;
 	char* password=NULL;
 	int   count;
